go/oasis-test-runner/oasis/network.go

```diff
@@ -136,6 +137,9 @@ type NetworkCfg struct { // nolint: maligned
 	// externally-provided.
 	UseShortGrpcSocketPaths bool `json:"-"`
 
+	// The runtime sandboxing provisioner. Can be `mock`, `unconfined` or `sandboxed`.
+	RuntimeProvisioner string `json:"-"`
```

Comment:
The runtime provisioner can already be configured per compute worker, why the need for global configuration? You can just leave the default fixture part, so that it is possible to configure the provisioner without rolling a custom fixture. And maybe add the runtime provisioner config to the client/keymanager node fixtures as well.